stl/src/xonce2.cpp

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+// __std_execute_once_begin & __std_execute_once_end functions

```

Comment:
Abbreviating "and" to "&" is confusing when we sometimes need to talk about references, bitwise AND, etc.

Also, the second function name is incorrect.

```suggestion
// __std_execute_once_begin and __std_execute_once_complete functions
```